src/WebJobs.Script/Binding/WebJobsCoreScriptBindingProvider.cs

```diff
@@ -147,130 +114,6 @@ public override Collection<Attribute> GetAttributes()
             }
         }
 
-        private class TableScriptBinding : ScriptBinding
-        {
-            public TableScriptBinding(ScriptBindingContext context) : base(context)
-            {
-            }
-
-            private string TableName
-            {
-                get { return Context.GetMetadataValue<string>("tableName"); }
-            }
-
-            private string PartitionKey
-            {
-                get { return Context.GetMetadataValue<string>("partitionKey"); }
-            }
-
-            private string RowKey
-            {
-                get { return Context.GetMetadataValue<string>("rowKey"); }
-            }
-
-            private string Filter
-            {
-                get { return Context.GetMetadataValue<string>("filter"); }
-            }
-
-            private int? Take
-            {
-                get { return Context.GetMetadataValue<int?>("take"); }
-            }
-
-            public override Collection<Attribute> GetAttributes()
-            {
-                Collection<Attribute> attributes = new Collection<Attribute>();
-
-                var attribute = new TableAttribute(this.TableName, this.PartitionKey, this.RowKey);
-                attribute.Filter = Filter;
```

Comment:
Definitely a risk. I've added more tests onto the SDK side for Table and Queue specifically